packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -65,7 +67,8 @@ export default async function setupAndLink(
 
   if (forceDelete) {
     const vercelDir = getVercelDirectory(path);
-    remove(vercelDir);
+    remove(join(vercelDir, VERCEL_DIR_README));
+    remove(join(vercelDir, VERCEL_DIR_PROJECT));
```

Comment:
Is this OK? I wasn't sure what the full intent of `forceDelete` was, but it was destroying all fixture setup under `.vercel`.

Reply:
I think this is fine. We only set this to true in the `vc link` command, to re-force the prompt.